qtawesome/tests/test_icon_browser.py

```diff
@@ -56,24 +56,34 @@ def test_copy(qtbot, browser):
     # Click the copy button
     qtbot.mouseClick(browser._copyButton, QtCore.Qt.LeftButton)
 
-    assert "google" in clipboard.text()
+    assert "penguin" in clipboard.text()
 
-
-def test_filter(qtbot, browser):
+def test_filter_success(qtbot, browser):
     """
-    Ensure the filter UX works
+    What happens if a `penguin`
     """
-    initRowCount = browser._listView.model().rowCount()
-    assert initRowCount > 0
+    #initRowCount = browser._listView.model().rowCount()
+    #assert initRowCount > 0
+
+    # Enter a search term and click
+    qtbot.keyClicks(browser._lineEditFilter, 'penguin')
+    qtbot.keyPress(browser._lineEditFilter, QtCore.Qt.Key_Enter)
+
+    filteredRowCount = browser._listView.model().rowCount()
+    assert filteredRowCount > 1
 
+def test_filter_fail(qtbot, browser):
+    """
+    What happens if a `not a penguin-penguin`
```

Comment:
Lets rewrite this to be:

```suggestion
    Ensure the filter doesn't show results if the filter text doesn't mach any icon
```